languages/elixir/exercises/concept/agent/.docs/after.md

```diff
@@ -0,0 +1,42 @@
+While spawning a process is easy, managing its state and behavior can become very complicated. The `Agent` module is an abstraction to facilitate this process to use an elixir process to manage a shared state.
+
+When using `Agent` module functions it's customary to encapsulate the agent-related functions in a single module.
+
+```elixir
+# A simple use of agent processes to represent a counter
+defmodule Counter do
+  def start_link(initial_value) do
+    Agent.start(fn -> initial_value end)
+  end
+
+  def value(pid) do
+    Agent.get(pid, & &1)
+  end
+
+  def increment(pid) do
+    Agent.update(pid, &(&1 + 1))
+  end
+end
+```
+
+The choice to use the `Agent` module, or to use multiple processes at all, depends on the relationship to be modelled with the data. _Agent processes_ are useful when a shared state must be used by multiple processes.
```

Comment:
I don't know where we stand on the whole AmE vs BrE vs CanE business, and I don't want to start a discussion... but I noticed that you spelled `modeled` in the introduction, and I like consistency 😇 
```suggestion
The choice to use the `Agent` module, or to use multiple processes at all, depends on the relationship to be modeled with the data. _Agent processes_ are useful when a shared state must be used by multiple processes.
```

Reply:
Canada being in a country between two spelling worlds, I often use either and don't even notice unless pointed out.  Thanks!